packages/ember-routing-htmlbars/lib/keywords/action.js

```diff
@@ -7,7 +7,7 @@ import { keyword } from 'htmlbars-runtime/hooks';
 import closureAction from 'ember-routing-htmlbars/keywords/closure-action';
 
 /**
-  The `{{action}}` helper provides a way to pass triggers for behavior (usually
+  The `{{action` helper provides a way to pass triggers for behavior (usually
```

Comment:
Ah, interesting. I think I've seen PRs where people added the closing `}}` thinking it was a typo. Either way, I agree with you that consistency is the most important thing.

Here's what I'd propose:

1. Use the unadorned `helper-name` (e.g. `action`) in prose: "The `action` helper provides a way to pass triggers for behavior."
2. Always include an example when introducing a new helper that shows it being invoked via the curly `{{action}}` syntax.

What do you think @jeradg @locks?